src/main/java/com/shika/pages/ACourseCategoryPage.java

```diff
@@ -47,10 +48,24 @@ public ACourseCategoryPage(WebDriver driver) {
 	@FindBy(how = How.ID, using = "toolbar-trash")
 
 	private WebElement trash;
+	
+	@FindBy(how = How.ID, using = "toolbar-batch")
+	
+	private WebElement batch;
+	
+    @FindBy(how = How.XPATH, using = "//*[@id=\"batch-access\"]")
+	
+	private WebElement setAccessLevel;

```

Comment:
name of Web Element variable shouldn't be generalised. Use "batchSetAccessLevel" instead.